samples/wifi/wfa_qt_app/README.rst

```diff
@@ -142,6 +142,9 @@ After programming, the sample shows the following output:
    [00:00:00.835,327] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
    [00:00:00.835,937] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
 
+.. note::
+   For running one Security Vulnerability Detection test case (Test case ID: 10003_2), please set Data Packet Type to ICMP in the Test Case Global Configuration section.
```

Comment:
```suggestion
   For running one Security Vulnerability Detection test case (Test case ID: 10003_2), set **Data Packet Type** to **ICMP** in the **Test Case Global Configuration** section of the QuickTrack Test Tool.
```

Reply:
Please remove "please".

Reply:
Removing this complete change as it not needed.